### PR TITLE
Exposing the 'stop_historu_eou_th' parameter

### PR DESCRIPTION
**Pull request by sarane22**
With these changes, clients can now configure `stop_eou_threshold` for streaming ASR:
The new parameters introduced are:
`--stop-eou-threshold = float`

### CODE REVIEW

**Comment by sarane22** [comment]
TODO: Update the `nvidia-riva/common` SHA once changes are merged.

File: riva/client/argparse_utils.py [hunk 1]
```diff
@@ -78,6 +78,12 @@ def add_asr_config_argparse_parameters(
         type=float,
         help="Threshold value for detecting the end of speech utterance",
     )
+    parser.add_argument(
+        "--stop-eou-threshold",
+        default=-1.0,
+        type=float,
+        help="Threshold value for likelihood of blanks before detecting end of utterance",
+    )
```
Comment (virajkarandikar) [inline riva/client/argparse_utils.py:86]: can you update help for `x-history` parameters to indicate the unit ? currently it doesn't mention. I assume it is in seconds or miliseconds ?

File: riva/client/asr.py [hunk 2]
```diff
@@ -130,6 +130,7 @@ def add_endpoint_parameters_to_config(
     stop_history: int,
     stop_history_eou: int,
     stop_threshold: float,
+    stop_threshold_eou: float,
 ) -> None:
     if not (start_history > 0 or start_threshold > 0 or stop_history > 0 or stop_history_eou > 0 or stop_threshold > 0):
```
Comment (rmittal-github) [inline riva/client/asr.py:135]: need to add `stop_threshold_eou` here

File: .gitmodules [hunk 3]
```diff
@@ -1,4 +1,4 @@
 [submodule "common"]
 	path = common
-	url = https://github.com/nvidia-riva/common.git
-	branch = main
+	url = https://github.com/sarane22/common.git
+	branch = endpointing_stop_eou_threshold_param
```
Comment (rmittal-github) [inline .gitmodules:4]: reminder comment to update this, once proto change is merged

**Review by rmittal-github — DISMISSED** [review]

File: riva/client/argparse_utils.py (hunk 1 above)
Comment (sarane22) [reply riva/client/argparse_utils.py:86]: Updated!

**Review by rmittal-github — APPROVED** [review]

**Pull request merged**